astropy/cosmology/flrw.py

```diff
@@ -415,7 +415,7 @@ def Om(self, z):
         """
 
         if isiterable(z):
-            z = np.asarray(z)
+            z=u.Quantity(z, u.dimensionless_unscaled).value
```

Comment:
Here and elsewhere, pep8 wants spaces around the equal sign...

Reply:
I've been going off the set of PEP8 style guides that Astropy checks...  We should expand this list to also include spacing around operators.

Reply:
I think we on purpose avoid spaces around operators, to allow things like `a*b + c`, but we definitely should have spaces around `=` for assignments.

Reply:
Done. Thanks @akeemlh